src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -979,7 +979,7 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
 
                     foreach (var item in items)
                     {
-                        using (var slicer = new TimeSeriesSliceHolder(context, docId, item.Name).WithBaseline(item.Baseline))
+                        using (var slicer = new TimeSeriesSliceHolder(context, docId, item.Name, collectionName.Name).WithBaseline(item.Baseline))
```

Comment:
@shiranshalom can you explain a bit more that change? Why we should / have to pass the collection here? How could it work without it?